src/msg/async/ProtocolV2.cc

```diff
@@ -94,6 +94,7 @@ ProtocolV2::ProtocolV2(AsyncConnection *connection)
       bannerExchangeCallback(nullptr),
       next_tag(static_cast<Tag>(0)),
       keepalive(false) {
+  ceph::crypto::init(cct);
```

Comment:
Just added this NSS-driven init. Currently we ship only `ceph::crypto::init` for NSS but it appears our builds have both `USE_NSS` and `USE_OPENSSL` defined while HMAC is only implemented with NSS. We will to change this, I guess.